### ISSUE TITLE
Sheet opens under UITabBarController

### ISSUE THREAD

**Issue by hugocarlmartin** [BeeMeeMan/FullScreenCover#3]
If i mix UIKit and SwifUI the sheet will actually open up under the tabBar. The question is, is it even possible to solve this?

**Comment by BeeMeeMan** [comment]
No, it is not. Cause this fullScreenCover not create another ViewController and it just cover another views in one VC.  So any created ViewController will overlay it.

To solve in SwiftUI:
 TabView {

}
.easyFullScreenCover(...)

**Comment by hugocarlmartin** [comment]
Well in my case its a UITabbarController

**Comment by loryhuz** [comment]
You can use .toolbar(.hidden, for: .tabBar) in your presented view
```
.easyFullScreenCover(isPresented: $showView) {
    NewView().toolbar(.hidden, for: .tabBar)
}
```
